po/de.po

```diff
@@ -847,7 +847,7 @@ msgstr "Terminalfenster"
 
 #: src/Gtk/SettingsWindow.vala:311
 msgid "Terminal command used to run"
-msgstr "Zur ausführung verwendeter terminalbefehl"
+msgstr "Terminal-befehl zur Ausführung"
 
```

Comment:
This does not seem correct, but I can't really know. I don't know German of course, but I know what the meaning of the string is supposed to be.

This is a fragment of a phrase that's concatenated with a variable to make the full phrase. And it's a label for a config setting.

I should really change so that it includes a %s, so that translators will know the full phrase, and can rearrange where the %s goes to make the proper structure for their language.

The full phrase is "Terminal command used to run \"%s\":"

The phrase is a label that basically says:  
This is the terminal command that will be used when we need to run something in a terminal: `foo ...`

You want to make a string that works as a label for that setting. In the settings screen, there is an input box after the string that contains the terminal command.

Altogether it looks like:  
Terminal command used to run mainline: `xterm -e`